storybook/storybook.tsx

```diff
@@ -8,7 +8,10 @@ configure(() => {
   require("./storybook-registry")
 }, module)
 
-const StorybookUI = getStorybookUI({ port: 9001, host: "localhost", onDeviceUI: true })
+const StorybookUI = getStorybookUI({
+  port: 9001, host: "localhost", onDeviceUI: true,
+  asyncStorage: require('@react-native-community/async-storage').default
```

Comment:
Is this the only place we use async storage?
Should we migrate to @react-native-async-storage/async-storage?

Reply:
No, we use async storage for saving MST snapshot

We can try to migrate `@react-native-async-storage/async-storage` in another PR